contactaccessrights/js/src/access-rights/controllers/access-rights-modal-ctrl.js

```diff
@@ -0,0 +1,113 @@
+define(['access-rights/modules/controllers'], function (controllers) {
+  'use strict';
+
+  controllers.controller('AccessRightsModalCtrl', ['Region', 'Location', 'Right', '$q', '$modalInstance',
```

Comment:
```js
controllers.controller('AccessRightsModalCtrl', [
    'dependency 1', 'dependency 2', 'dependency 3',
    function (1, 2, 3) {

    }
]);
```
(it's future-friendly if we'll need to break line when we have many injected dependencies)